math/min/bitwisemin.go

```diff
@@ -0,0 +1,12 @@
+// bitwisemin.go
+// description: Gives min of two integers
+// details:
+// implementation of finding the minimum of two numbers using only binary operations without using conditions
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see bitwisemin_test.go
+
+package min
+
+func BitwiseMin(a int, b int, base int) int {
+	return a&((a-b)>>base) | b&(^(a-b)>>base)
+}
```

Comment:
I believe it will be as straight forward as this

```suggestion
func Bitwise(base int, vals ...int) int {
	temp := a[0]
	for _, val := range vals {
		temp = temp&((temp-val)>>base) | val&(^(temp-val)>>base)
	}
	return a&((a-b)>>base) | b&(^(a-b)>>base)
}
```

there is only one extra assignment but that is negligible IMO. Haven't tested it so maybe test it in your local env and see if this works 😄

Reply:
Done =)